scripts/dev-bundle-tool-package.js

```diff
@@ -41,30 +41,30 @@ var packageJson = {
     "source-map": "0.7.4",
     chalk: "4.1.2",
     // TODO: maybe replace with https://www.npmjs.com/package/better-sqlite3
-    sqlite3: "5.0.2",
+    sqlite3: "5.1.7",
     inquirer: "8.2.6",
     "http-proxy": "1.18.1",
     "is-reachable": "3.1.0",
     "wordwrap": "1.0.0",
-    "moment": "2.29.1",
+    "moment": "2.30.1",
     "rimraf": "2.6.2",
     "glob": "7.1.6",
-    ignore: "3.3.7",
+    ignore: "5.3.2",
```

Comment:
This ignore tool dependency is used for .meteorignore file formatting, there are extra features for latest version, and according to my research doesn't produce any breaking change. But I will do further testing.